src/Deprecated.jl

```diff
@@ -5,6 +5,11 @@
 ##==============================================================================
 ## Remove in 0.11
 ##==============================================================================
+
+export listSolvekeys
```

Comment:
FIY @deprecate automatically exports the name

Reply:
oh, didnt know that thanks!